app/views/guilds/show.html.erb

```diff
@@ -3,6 +3,12 @@
 <%# TABS BEGIN %>
 <div class="main-container">
   <div class="container mt-5">
+    <div data-controller="clipboard" class="d-flex flex-row-reverse">
+    <%# if guild.owner == current_user do %>
+      PIN: <input data-clipboard-target="source" type="text" value='http://localhost:3000/invite/<%=@guild.join_code%>' readonly class="mb-5 d-none">
```

Comment:
This will have to be changed before a production push, but don't worry about it for the moment